pkg/secrethub/credentials/key.go

```diff
@@ -85,6 +85,9 @@ func ImportKey(credentialReader, passphraseReader Reader) (Key, error) {
 		if envPassphrase != "" {
 			credential, err := decryptKey([]byte(envPassphrase), encoded)
 			if err != nil {
+				if crypto.IsWrongKey(err) {
+					err = ErrCannotDecryptCredential
+				}
 				return Key{}, fmt.Errorf("decrypting credential with passphrase read from $%s: %v", credentialPassphraseEnvVar, err)
```

Comment:
Is it useful to include the source of the passphrase in this error message as well?

Reply:
I'm not sure if I get you correctly. Because I think that already happens.

Reply:
Sorry, all is good :+1:

I misread this line `err = ErrCannotDecryptCredential` and thought the error was returned there. :see_no_evil: